CalibTracker/SiPixelESProducers/BuildFile.xml

```diff
@@ -6,7 +7,10 @@
 <use name="FWCore/Framework"/>
 <use name="FWCore/MessageLogger"/>
 <use name="FWCore/ParameterSet"/>
+<use name="HeterogeneousCore/AlpakaCore"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
 <use name="HeterogeneousCore/CUDACore"/>
+<flags ALPAKA_BACKENDS="1"/>
```

Comment:
After moving the `TYPELOOKUP_DATA_REG()` registrations to their proper package, there should be no need to change anything in this `BuildFile.xml`.